Windows gradle_plugin_light_apk_test is 2.00% flaky

<!-- meta-tags: To be used by the automation script only, DO NOT MODIFY.
{
  "name": "Windows gradle_plugin_light_apk_test"
}
-->

The post-submit test builder `Windows gradle_plugin_light_apk_test` had a flaky ratio 2.00% for the past (up to) 100 commits, which is above our 2.00% threshold.

One recent flaky example for a same commit: https://ci.chromium.org/ui/p/flutter/builders/prod/Windows%20gradle_plugin_light_apk_test/5197
Commit: https://github.com/flutter/flutter/commit/6aed693c4c961c6f3c16703ea231d01497812095

Flaky builds:
https://ci.chromium.org/ui/p/flutter/builders/prod/Windows%20gradle_plugin_light_apk_test/5197
https://ci.chromium.org/ui/p/flutter/builders/prod/Windows%20gradle_plugin_light_apk_test/5140

Recent test runs:
https://flutter-dashboard.appspot.com/#/build?taskFilter=Windows%20gradle_plugin_light_apk_test

Please follow https://github.com/flutter/flutter/wiki/Reducing-Test-Flakiness#fixing-flaky-tests to fix the flakiness and enable the test back after validating the fix (internal dashboard to validate: go/flutter_test_flakiness).

The flake bot only auto deflake a test when the corresponding flaky bug is closed (either fixed or not reproducible). As this bug was still open so the bot skipped the test.

Looking through the data of this test, seems one more flake was hit on https://github.com/flutter/flutter/commit/0b5032a058622878fbac7cf41d3a4e7ab276e574, with flaky build: https://ci.chromium.org/ui/p/flutter/builders/staging/Windows%20gradle_plugin_light_apk_test/177/overview

```
[gradle_plugin_light_apk_test] [STDOUT] Executing "C:\b\s\w\ir\x\w\recipe_cleanup\tmplj95wc1z\flutter sdk\bin\flutter build apk --release --target-platform=android-arm64" in "C:\b\s\w\ir\x\t\flutter_devicelab_gradle_plugin_test.22e638ad\aaa\example\android" with environment {BOT: true, LANG: en_US.UTF-8}
[gradle_plugin_light_apk_test] [STDOUT] stdout: Changing current working directory to: C:\b\s\w\ir\x\t\flutter_devicelab_gradle_plugin_test.22e638ad\aaa\example
[gradle_plugin_light_apk_test] [STDOUT] stdout: 
[gradle_plugin_light_apk_test] [STDOUT] stdout:  Building with sound null safety 
[gradle_plugin_light_apk_test] [STDOUT] stdout: 
[gradle_plugin_light_apk_test] [STDOUT] stdout: Running Gradle task 'assembleRelease'...                        
[gradle_plugin_light_apk_test] [STDOUT] stderr: Warning: Mapping new ns http://schemas.android.com/repository/android/common/02 to old ns http://schemas.android.com/repository/android/common/01
[gradle_plugin_light_apk_test] [STDOUT] stderr: Warning: Mapping new ns http://schemas.android.com/repository/android/generic/02 to old ns http://schemas.android.com/repository/android/generic/01
[gradle_plugin_light_apk_test] [STDOUT] stderr: Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/addon2/02 to old ns http://schemas.android.com/sdk/android/repo/addon2/01
[gradle_plugin_light_apk_test] [STDOUT] stderr: Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/repository2/02 to old ns http://schemas.android.com/sdk/android/repo/repository2/01
[gradle_plugin_light_apk_test] [STDOUT] stderr: Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/sys-img2/02 to old ns http://schemas.android.com/sdk/android/repo/sys-img2/01
[gradle_plugin_light_apk_test] [STDOUT] stderr: Dart snapshot generator failed with exit code -1073740940
[gradle_plugin_light_apk_test] [STDOUT] stderr: 
[gradle_plugin_light_apk_test] [STDOUT] stderr: 
[gradle_plugin_light_apk_test] [STDOUT] stderr: FAILURE: Build failed with an exception.
[gradle_plugin_light_apk_test] [STDOUT] stderr: 
[gradle_plugin_light_apk_test] [STDOUT] stderr: * Where:
[gradle_plugin_light_apk_test] [STDOUT] stderr: Script 'C:\b\s\w\ir\x\w\recipe_cleanup\tmplj95wc1z\flutter sdk\packages\flutter_tools\gradle\flutter.gradle' line: 1177
[gradle_plugin_light_apk_test] [STDOUT] stderr: 
[gradle_plugin_light_apk_test] [STDOUT] stderr: * What went wrong:
[gradle_plugin_light_apk_test] [STDOUT] stderr: Execution failed for task ':app:compileFlutterBuildRelease'.
[gradle_plugin_light_apk_test] [STDOUT] stderr: > Process 'command 'C:\b\s\w\ir\x\w\recipe_cleanup\tmplj95wc1z\flutter sdk\bin\flutter.bat'' finished with non-zero exit value 1
[gradle_plugin_light_apk_test] [STDOUT] stderr: 
```

Reopening to track the flake.

--- a/.ci.yaml
+++ b/.ci.yaml
@@ -2770,30 +2770,6 @@ targets:
       - bin/**
       - .ci.yaml
 
-  - name: Mac gradle_plugin_light_apk_test
-    recipe: devicelab/devicelab_drone
-    timeout: 60
-    properties:
-      caches: >-
-        [
-          {"name":"gradle","path":"gradle"}
-        ]
-      dependencies: >-
-        [
-          {"dependency": "android_sdk", "version": "version:31v8"},
-          {"dependency": "open_jdk"},
-          {"dependency": "xcode"},
-          {"dependency": "gems"}
-        ]
-      tags: >
-        ["devicelab","hostonly"]
-      task_name: gradle_plugin_light_apk_test
-    scheduler: luci
-    runIf:
-      - dev/**
-      - bin/**
-      - .ci.yaml
-
   - name: Mac module_custom_host_app_name_test
     recipe: devicelab/devicelab_drone
     timeout: 60
@@ -3936,30 +3912,6 @@ targets:
         ["devicelab","hostonly"]
       task_name: gradle_plugin_fat_apk_test
     scheduler: luci
-
-  - name: Windows gradle_plugin_light_apk_test
-    bringup: true # Flaky https://github.com/flutter/flutter/issues/97956
-    recipe: devicelab/devicelab_drone
-    timeout: 60
-    properties:
-      caches: >-
-        [
-          {"name":"gradle","path":"gradle"}
-        ]
-      dependencies: >-
-        [
-          {"dependency": "android_sdk", "version": "version:31v8"},
-          {"dependency": "chrome_and_driver", "version": "version:84"},
-          {"dependency": "open_jdk"}
-        ]
-      tags: >
-        ["devicelab","hostonly"]
-      task_name: gradle_plugin_light_apk_test
-    scheduler: luci
-    runIf:
-      - dev/**
-      - bin/**
-      - .ci.yaml
 
   - name: Windows hot_mode_dev_cycle_win_target__benchmark
     recipe: devicelab/devicelab_drone

